docs/en/enterprise-edition/policy-reference/azure-policies/azure-general-policies/bc-azure-237.adoc

```diff
@@ -1,5 +1,5 @@
 
-== Dedicated data endpoint is disabled
+== Azure Container Registry dedicated data endpoint is disabled
 
```

Comment:
Disabled dedicated Azure Container Registry endpoint

Reply:
We try to lead with the service